resources/user-guides/launch-data-elements.md

```diff
@@ -0,0 +1,86 @@
+
+Data elements are populated with data when they are processed within the Experience Platform Mobile SDK [Rules Engine](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine). To use data elements, at a high level you can follow these steps:
+
+1. Create a data element.
+2. Use the data element when defining a rule.
+3. [Publish](https://docs.adobe.com/content/help/en/launch/using/reference/publish/overview.html) your property, which publishes the rules JSON and make it available for download by your application(s).
+
+{% hint style="info" %}
+
+When a new Extension is added to your property, new data elements may become available to use. You will need to repeat the steps above to create new data elements in order to use them when creating new rules.
+
+{% endhint %}
+
+## Create a data element
+
+Data elements are building blocks for rules. Data elements let you create a data dictionary of commonly used data in the Experience Platform Mobile SDK, regardless of where they originate (shared state, event data) or which extension creates them.
+
+1. From a Property page, open the Data Elements tab, then click Add Data Element.
+
+2. Give a unique, descriptive name to the data element.
+
+3. Select the Extension which generates the data element type. 
+
+   You may only select Extensions which are currently installed in your property.
+
+4. Select the Data Element Type. 
+
+   Add the required configuration parameters. Most data element types do not require any configuration.
+
+5. Click Save.
+
+For example, to create a data element which maps to the Experience Cloud ID, from the Create New Data Element page, select Extension *Mobile Core*, Data Element Type *Experience Cloud ID* and enter Name *ECID*. 
+
+![create ECID data element](../../.gitbook/assets/data-elements-create-data-element-ecid.png)
+
+## Data Element Usage
+
+You can use data elements when defining rules in the Experience Platform Launch interface. When a rule configuration allows the use of data elements, a cylinder icon ![cylinder icon](../../.gitbook/assets/data-elements-cylinder.png) is displayed next to the configuration text field. Clicking the icon gives a list of data elements defined in the mobile property.
+
+![data element selection](../../.gitbook/assets/data-elements-select-data-element.png)
+
+Here is an example which creates a rule to send a postback containing the Experience Cloud ID when the application launches.
+
+1. From a Property page, open the Rules tab, then click Create New Rule.
+
+   ![create rule](../../.gitbook/assets/data-elements-create-rule.png)
+
+2. Give a unique, descriptive name to the rule.
+
+3. Add an Event configuration. 
+
+   Select Extension *Mobile Core* and Event Type *Launched*.
+
+   ![event configuration](../../.gitbook/assets/data-elements-event-configuration.png)
+
+4. Add a Condition configuration. 
```

Comment:
Please reword this section as follows:
"6. In the **Condition** section, click **Add**.
7. In the **Extension** drop-down list, select **Mobile Core**.
8. In the **Condition Type** drop-down list, select **Data Element**.
9. Enter a name for the condition. (this step appears to be missing in the original text).
9. Next to the **Data Element** text field, click the cylinder icon and select the ECID that was created in the previous section.